project/Dependencies.scala

```diff
@@ -94,14 +95,16 @@ object Dependencies {
     "org.scalacheck" %% "scalacheck" % versions.scalaCheckVersion % "test",
     "javax.xml.bind" % "jaxb-api" % versions.jaxbVersion,
     "com.github.alexarchambault" %% "scalacheck-shapeless_1.13" % versions.scalaCheckShapelessVersion % "test"
-  ) ++ awsDependencies ++ akkaDependencies ++ dynamoDependencies ++ jacksonDependencies ++ circeDependencies
+  ) ++ awsDependencies ++ akkaDependencies ++ dynamoDependencies ++ circeDependencies
 
   val pipelineModelDependencies = circeDependencies
 
   val commonDisplayDependencies: Seq[ModuleID] = swaggerDependencies
 
   val commonElasticsearchDependencies = commonDependencies ++ esDependencies
 
+  val commonMessagingDependencies = commonDependencies ++ jacksonDependencies
```

Comment:
Actually, why does messaging depend on jackson? I thought we always use circe for messaging..

Reply:
I’ll add a comment – we’re using jackson-databind to parse YAML in the SNS fixture.

Reply:
pff, I see

Reply:
Maybe we should be using [this](https://github.com/circe/circe-yaml) (Not in this PR, but link it in the comment?)

Reply:
Yep, standardising on Circe would be good. In the comment.